src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,199 @@
+import cn from 'classnames';
```

Comment:
Something wrong with input on form, it clears input value when i try to write.
https://www.loom.com/share/5715b5ecb8d24efc9fee377690a57401

Reply:
I don't know why and how it works, but this problem only works on your demo.
When I write "asd" fast enough.
https://www.loom.com/share/8225319209c3459bb2769f00993985c3

Reply:
It clears an input when I write a message similar to last "todo" title
https://www.loom.com/share/7a9f872263874f6c9483ab2ef97fb933

Reply:
It clears `input` when I try to write a message similar to the last "todo" title

Reply:
https://www.loom.com/share/7a9f872263874f6c9483ab2ef97fb933